### PR TITLE
doc: document new scrub-related 'pg dump' columns

### PR DESCRIPTION
**Pull request by ronen-fr**
Signed-off-by: Ronen Friedman <rfriedma@redhat.com>

### CODE REVIEW

**Comment by ronen-fr** [comment]
related to https://github.com/ceph/ceph/pull/43403

File: PendingReleaseNotes [hunk 1]
```diff
@@ -73,6 +73,11 @@
 
   https://docs.ceph.com/en/latest/rados/operations/placement-groups/
 
+* The ``pg dump`` command now prints two additional columns:
```
Comment (neha-ojha) [inline PendingReleaseNotes:76]: ceph pg dump

**Review by neha-ojha — APPROVED** [review]
It might make sense to add this commit to https://github.com/ceph/ceph/pull/43403, so that everything gets backported together

File: PendingReleaseNotes (hunk 1 above)
Comment (ronen-fr) [reply PendingReleaseNotes]: fixed

**Comment by ronen-fr** [comment]
> It might make sense to add this commit to #43403, so that everything gets backported together

@neha-ojha: Added to #43403.

**Comment by neha-ojha** [comment]
> > It might make sense to add this commit to #43403, so that everything gets backported together
> 
> @neha-ojha: Added to #43403.

Thanks!

**Comment by neha-ojha** [comment]
Included in https://github.com/ceph/ceph/pull/43403

**Pull request closed without merging**